### DISCUSSION TITLE
App crash in TrackbasedConferenceActivity.java

### DISCUSSION THREAD

**Discussion by AndroidApprikart** [ant-media/Ant-Media-Server#5534] [Q&A]
With the latest SDK, the sample app crashes when the audio and video is toggled in the TrackbasedConferenceACtivity with 2 participants in the room. 

The now deprecated MultitrackConference Activity works fine in this case.

**Comment by yashtandon113** [comment]
@AndroidApprikart 
As discussed [here](https://github.com/orgs/ant-media/discussions/5468), Multitrack conference application works fine while the old conference application does not.
So we recommend you to use the MultiTrack conference activity.
